tests/foreman/ui/test_computeresource_gce.py

```diff
@@ -87,8 +89,6 @@ def test_positive_default_end_to_end_with_custom_profile(
         assert cr_values['provider_content']['http_proxy']['value'] == http_proxy.name
         assert cr_values['organizations']['resources']['assigned'] == [module_org.name]
         assert cr_values['locations']['resources']['assigned'] == [module_location.name]
```

Comment:
You can assert json_key here.

Reply:
json_key has multiple values to assert and our main params are zone and the whole key itself, the zone is getting asserted here hence thinking not to go for the whole assertion as if the data values are incorrect the CR will not be formed anyway.

Reply:
In that case, you can skip these assertion.